Applications/DataExplorer/mainwindow.cpp

```diff
@@ -425,15 +425,18 @@ void MainWindow::save()
     }
     else if (fi.suffix().toLower() == "geo")
     {
-        int const return_val =
-            FileIO::GMSHInterface::writeGeoFile(_project.getGEOObjects(), fileName.toStdString());
+        std::vector<std::string> selected_geometries;
+        _project.getGEOObjects().getGeometryNames(selected_geometries);
 
-        if (return_val == 1)
+        GMSHInterface gmsh_io(
+            _project.getGEOObjects(), true,
+            FileIO::GMSH::MeshDensityAlgorithm::AdaptiveMeshDensity,
+            0.05, 0.5, 2, selected_geometries);
```

Comment:
So we just assume these values are correct when writing a geo-file? I just want to understand why specific parameter values are given when that wasn't necessary before. Because the resulting geo-file will be quite specific concerning the mesh that will would be created based on it.